lib/pure/collections/sets.nim

```diff
@@ -571,10 +571,23 @@ proc map*[A, B](data: HashSet[A], op: proc (x: A): B {.closure.}): HashSet[B] =
   result = initHashSet[B]()
   for item in items(data): result.incl(op(item))
 
+from std/algorithm import sort
```

Comment:
Still I prefer the old solution, sorting is expensive, depedencies between stdlib modules should be minimal and the real bug here was that ``items`` wasn't used.

Reply:
I'm now using a different algorithm that doesn't require `sort` nor allocations